tests_e2e/GuestAgentDcrTestExtension/GuestAgentDcrTest.py

```diff
@@ -4,11 +4,18 @@
 
 from Utils.WAAgentUtil import waagent
 import Utils.HandlerUtil as Util
+import datetime
 import sys
 import re
 import traceback
 import os
-import datetime
+
+if sys.version_info[0] == 3 and sys.version_info[1] >= 11:
```

Comment:
I used a simple approach for the test extension, which has only 1 single call to utcnow